test/suites/dev/test-filter/test-filter-api-pending.ts

```diff
@@ -9,6 +9,8 @@ describeSuite({
     it({
       id: "T01",
       title: "should not be supported",
+      // Looks like this is now supported 🎉
+      modifier: "skip",
```

Comment:
This is equivalent to `it.skip()`? Or something else?

Also, why are we skipping this now?

Reply:
`it.skip()` is a mocha feature which isn't supported in moonwall (since mocha has been ditched).

Moonwall instead uses a optional param.

We are skipping this test case for now because we used to test that this RPC was disabled - however latest FP has enabled this endpoint.